src/GameManager.cpp

```diff
@@ -1,3 +1,48 @@
 #include "GameManager.hpp"
 
 DIFFABLE_STATIC(GameManager, g_GameManager);
+
+#pragma optimize("s", on)
+GameManager::GameManager()
+{
+
+    memset(this, 0, sizeof(GameManager));
+
+    (this->arcadeRegionTopLeftPos).x = 32.0;
```

Comment:
I feel like we should put some #defines with those values somewhere, to make it easier to tweak later later. Something like `GAME_REGION_{TOP,LEFT,RIGHT,BUTTOM}`

Reply:
Shouldn't be something more like GAME_REGION_{TOP, LEFT}, and then GAME_REGION_{WIDTH, HEIGHT}?

Reply:
Ah yes, you're right.